canonical-web-and-design/charmhub.io
# Add docs tab

## Done

- Add docs tab

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045http://0.0.0.0:8045/charm#docs
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e2abd081ad403548e9afe0d)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1267

## Note

- For now the side navigation is implemented as vanilla `table of contents`
- We need to discuss at the next charmhub meeting weather we use `table of contents` or the new `side navigation` for `docs` and `configuration` tabs
- Navigation from the table of contents links does not work with `js`
